packages/backend-lib/src/server/createDefaultApp.ts

```diff
@@ -121,7 +121,7 @@ export async function createDefaultApp(cfg: DefaultAppCfg): Promise<BackendAppli
   // Generic error handler
   // It handles errors, returns proper status, does sentry.captureException(),
   // assigns err.data.errorId from sentry
-  app.use(genericErrorMiddleware({ sentryService, ...cfg.genericErrorMwCfg }))
+  app.use(genericErrorMiddleware({ errorService: sentryService, ...cfg.genericErrorMwCfg }))
```

Comment:
I'm ok if you want to make this PR smaller and keep it named `sentryService` here (in `DefaultAppCfg` and `StartServerCfg`.

But I just checked these 2 upstream interfaces - and they should work if we **do** the rename to `errorReportingService` and make them accept `ErrorReportingService`.

... or wait. Actually, `startServer` needs more than `ErrorReportingService`.

TLDR: super fine to keep things as is. You can change to `errorReportingService` in `createDefaultApp` safely. But not in `startServer`, as that one needs more thinking 🙏

Reply:
But since startServer's configuration extends the createDefaultApp's configuration, doing the change only in one of them looks confusing.